app.js

```diff
@@ -12,21 +12,27 @@ const models = path.join(lib, 'models.js');
 
 const {Track} = require(models);
 
+const Configfile = require('electron-config');
+const configfile = new Configfile();
```

Comment:
A really minor thing, could you change the name to configFile thus respecting the camel case rule?